src/engine/enginemaster.h

```diff
@@ -252,6 +244,13 @@ class EngineMaster : public QObject, public AudioSource {
                              sizeof(long double)];
     };
 
+    std::atomic<MacroState>* m_pMacroState;
+    Macro* m_pMacroRecording;
```

Comment:
make these private

Reply:
oh I see EngineBuffer has to do an atomic compare_exchange with the recording state